### DISCUSSION TITLE
Future of Palm 🌴 🚀

### DISCUSSION THREAD

**Discussion by norton120** [palmetto/palm-cli#106] [General]
Is is probably as good a time as ever to consider the maintainer strategy for 2024 and onward. Two things I think we should square away:

- **makeup of core team**
- **general direction**

For **makeup of core team**:
I know I rely on Palm pretty heavily for almost all my workflows, which means:
- improvements to Palm have a multiplier effect across my work
- any regressions/bugs have a similar multiplier

So it behooves me to set a contribution schedule and get back into active development. 

I'm assuming a few other people are in the same boat, and so it would be great to discuss who wants to be on the core team (and if we can pay @JakeBeresford in gift cards from the Old City fishmonger to stick around). 

For **general direction**:
Palm started out focused on supporting data-analytics teams, because that was where we were (and it was the right fit of Python, mixed OSs, and a need to simplify cli work for SQL developers). 
I personally use Palm on every project now - and that ranges from web applications to AI/generative to data pipelines. Is it still "A toolbelt for data teams?" or is it fast, easy, modular workflow abstraction for _any_ kind of software?

**Comment by JakeBeresford** [comment]
Thanks for raising this discussion @norton120. 

I am glad to hear that youa re making use of Palm in your projects and that it is adding value for you & your team.

You can rest assured that I am still a maintainer on this repo and will continue to review PRs, address bug reports, and ensure this project remains available and up-to-date for users like yourself. I will accept donations in the form of seafood gift cards and/or high quality rum.
